torch/tensor.py

```diff
@@ -425,8 +425,13 @@ def __contains__(self, element):
         """
         if isinstance(element, (torch.Tensor, Number)):
             return (element == self).any().item()
-        return NotImplemented
 
+        raise RuntimeError(
+            "Tensor.__contains__ only supports scalars, but you passed in a %s. "
+            "Please file a bug if you believe this message to be incorrect." %
```

Comment:
I also don't think we should have this -- people should file bugs if any error message is incorrect, I don't think we need to spell that out.

Reply:
ATen's contributors like to throw such an error message in a CPP file:
```
template <typename T>
void badArgType(const T& v) {
  AT_ERROR(
      "Found an unsupported argument type in the JIT tracer: ",
      c10::demangle_type<T>(),
      ". File a bug report.");
}
```
But I do think we could make the error message more streamlined in the common operations.